docker-compose.yml

```diff
@@ -128,6 +128,8 @@ services:
       KONG_DATABASE: "postgres"
       KONG_CASSANDRA_CONTACT_POINTS: "cassandra"
       KONG_PG_HOST: "postgres"
+    volumes:    
+      - ./apigw/plugins/pep-kong:/plugins/pep-kong
```

Comment:
Isn't just '/apigw/plugins/'?
If we need to add more plugins, more volumes must be included?

Reply:
The leading period is important to tell docker-compose that this is a relative path. Setting the local path (i.e. the one prior to the colon `:`) would make docker compose search for the folder on the root directory, which would (most likely) fail.

It was my understanding that yes, for each plugin, a new volume would be passed down to the container.

Reply:
Each plugin uses its own volume inside /plugins/<plugin_name>. That way the developer have the controll over which plugin is installed and, in case they are developing a new plugin, where that plugin is located without the need of creating symbolic links.
The default dojot plugins will be listed in this repository.